presto-orc/src/main/java/com/facebook/presto/orc/writer/TimestampColumnWriter.java

```diff
@@ -161,9 +162,12 @@ public void writeBlock(Block block)
 
                 secondsStream.writeLong(seconds);
                 nanosStream.writeLong(encodedNanos);
-                nonNullValueCount++;
+                blockNonNullValueCount++;
             }
         }
+
+        nonNullValueCount += blockNonNullValueCount;
+        return (block.getPositionCount() - blockNonNullValueCount) * NULL_SIZE + blockNonNullValueCount * (Long.BYTES + Integer.BYTES);
```

Comment:
I'd extract `(Long.BYTES + Integer.BYTES)` into a constant and add a comment why we use INT size for the nanos field despite using `LongOutputStreamV1.writeLong` for both seconds and nanos.

Reply:
Done.